traffic_ops/client/session.go

```diff
@@ -369,12 +368,18 @@ func (to *Session) RawRequest(method, path string, body []byte, header http.Head
 		if err != nil {
 			return nil, remoteAddr, err
 		}
+		if header != nil {
```

Comment:
Nitpick: these nil checks aren't necessary, because `http.Header.Clone` is defined to: 
> Clone returns a copy of h or nil if h is nil.

Reply:
No, they are, because if it returns nil I can't do
```go
req.Header = nil
```
then
```go
req.Header.Set("something", "something else")
```
will segfault.